src/Sylius/Behat/Context/Ui/Admin/ManagingPromotionsContext.php

```diff
@@ -107,6 +109,23 @@ public function iAddIt()
         $this->createPage->create();
     }
 
+    /**
+     * @When I specify its label as :label in :localeCode locale
+     */
+    public function iSpecifyItsLabelInLocaleCode(string $label, string $localeCode): void
+    {
+        $this->createPage->specifyLabel($label, $this->localeConverter->convertNameToCode($localeCode));
+    }
+
+    /**
+     * @When the :promotion promotion should have a label :label in :locale locale
```

Comment:
```suggestion
     * @When the :promotion promotion should have a label :label in :localeCode locale
```
and the usage of `LocaleConverterInterface::convertNameToCode` should not be needed